src/app/(home)/page.tsx

```diff
@@ -219,21 +126,25 @@ function Home() {
           ))}
         </h2>
         <h3 className="mt-3 sm:mt-5 text-zinc-800 text-2xl sm:text-4xl font-light">
-          <Trans i18nKey="description" ns="index">
+          <Trans i18n={i18n} i18nKey="description" ns="index">
             <strong className="font-medium">xLog</strong> is the best{" "}
-            <UniLink
+            <a
```

Comment:
IMHO, In RSC, we shouldn't use components that may be both pure function component and stateful component (next/link) like `<UniLink/>`, so I just replace to anchor tag here